src/meetup/MeetupTitle.tsx

```diff
@@ -1,22 +1,75 @@
 import {AbsoluteFill, spring, useCurrentFrame, useVideoConfig} from 'remotion';
 import {Title} from '../components/Title';
+import {LyonJSLogo} from '../components/LyonJSLogo';
 
 export const MeetupTitle: React.FC<{title: string}> = ({title}) => {
 	const frame = useCurrentFrame();
 	const {fps, width} = useVideoConfig();
 
-	const left = spring({frame, from: -width, to: 0, fps, durationInFrames: 30});
+	const scaleUpLogo = spring({
+		frame,
+		from: 0,
+		to: 1,
+		fps,
+		durationInFrames: 30,
+	});
+	const scaleDownLogo = spring({
+		frame: frame - 50,
+		from: 0,
+		to: 1,
+		fps,
+		config: {
+			mass: 0.6,
+			damping: 5,
+			stiffness: 50,
+		},
+		durationInFrames: 65,
+	});
+
+	const fromBottom = spring({
+		frame,
+		from: 0,
+		to: 300,
+		fps,
+		durationInFrames: 30,
+	});
+	const toBottom = spring({
+		frame: frame - 50,
+		from: 0,
+		to: 500,
+		fps,
+		durationInFrames: 40,
+	});
+
+	const opacityUp = spring({frame, from: 0, to: 1, fps, durationInFrames: 50});
+	const opacityDown = spring({
+		frame: frame - 50,
+		from: 0,
+		to: 0.5,
+		fps,
+		durationInFrames: 70,
+	});
```

Comment:
Maybe we could split it in two sub component and pass the `50` value as props of those component ?